Automattic/wp-calypso
# Plans: Roll out Monthly pricing globally

#### Changes proposed in this Pull Request

* Roll out Monthly pricing globally

#### Testing instructions



_Signup Process_

1. Visit `/start` to begin the signup process.
2. Make sure monthly plans appear in the plans step.
  <img width="800" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/104121999-c4a78200-5385-11eb-905b-6fe845c18e23.png">
  <img width="528" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/104122016-d9841580-5385-11eb-806b-f1c929aac212.png">
3. In the checkout page, the summary should show what monthly plans don't provide.
  <img width="954" alt="Checkout_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/104122077-2e279080-5386-11eb-921d-d8389a0f326d.png">

_My site_

1. The Plans page should show monthly plans when your site is on a monthly plan.
  <img width="1260" alt="Plans_‹_Site_Title_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/104122697-e276e600-5389-11eb-8d02-75dd518d558a.png">

2. The My Plan also has to display "monthly" next to the current plan name.
<img width="800" alt="My_Plan_‹_Site_Title_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/104122768-587b4d00-538a-11eb-982d-75101d25dc1c.png">
